tests/integration/helpers/utils.py

```diff
@@ -23,3 +32,135 @@ def clear_zk_children(zk_client, path):
             zk_client.delete(path + '/' + node)
     finally:
         close_zk_client(zk_client)
+
+
+class MultiReadRequest(object):
+    def __init__(self, client):
```

Comment:
If ` MultiReadRequest` is based on python zookeeper lib, please add a comment.

Reply:
okay, I will add some comment for `MultiReadRequest`.